src/main/java/SingleDriver.java

```diff
@@ -0,0 +1,38 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class SingleDriver {
+    private static WebDriver driver;
```

Comment:
Here must be instance of singleton class type

Reply:
This was added